### ISSUE TITLE
Questions about training process

### ISSUE THREAD

**Issue by Yingjuan** [estenhl/pyment-public#15]
Dear authors, 

Thank you for providing the models and associated scripts! 
I have successfully completed data preprocessing following "/notebooks/Download and preprocess IXI.ipynb". Now I am trying to train the model based on my data according to "/notebooks/Train brain age model.ipynb". 
I have two questions regarding the training process:
1. It appears that the provided script trains the model from scratch. If I intend to initialize the training with pre-trained weights, could you kindly guide me on the appropriate steps to take?
2. The 'input_shape' in the script is set as [43, 54, 41], while my preprocessed data have dimensions of 167x212x160. Did I miss some steps in between or is it necessary to modify the 'input_shape' to match the dimensions of my preprocessed images?

Your guidance would be greatly appreciated. Thank you in advance for your help!

Yingjuan

**Comment by estenhl** [comment]
Hi!

If you want to finetune the model from our pretrained weights you could follow the "Train sex-classifier using transfer learning" notebook in the dev-branch. If you just use the model called "backbone" from the 2nd code block  you should be good to go. And yes, the input_shape should be set to match your images :)

**Comment by Yingjuan** [comment]
Thank you for your prompt response! I appreciate your guidance as I am relatively new to this field.

I have two more questions regarding fine-tuning the model for age prediction using pretrained weights:
1. Should I retain the existing top layer of the pretrained model, or is it advisable to add a new dense layer, like in the provided code below?
2. My dataset includes participants aged over 99, which exceeds the prediction range of the pretrained model. Would it be sufficient to add "prediction_range=[50,105]" to accommodate this extended age range?
backbone = RegressionSFCN(input_shape=[167, 212, 160], weights='brain-age', include_top=False, \
                          dropout=0.5, weight_decay=1e-3, prediction_range=[50,105])
head = Dense(1, activation=None)(backbone.output)
model = Model(backbone.input, head)

Thank you so much!

**Comment by estenhl** [comment]
Hi again, sorry for the delay this time. You don't need the additional dense layer if you want to do age regression, in fact you shouldn't need to make any changes at all :) And yeah, modifying the prediction range should work off-the-shelf like that. Let me know whether it works!

**Comment by Yingjuan** [comment]
Thank you for the reply! Since I have limited expertise, would you kindly provide me with your guidance to clarify which option I should choose to fine-tune the model for age regression? 
1.
backbone = RegressionSFCN(input_shape=[167, 212, 160], weights='brain-age', include_top=**False**,
dropout=0.5, weight_decay=1e-3, prediction_range=[50,105])
head = Dense(1, activation=None)(backbone.output)
model = Model(backbone.input, head)

2.
model = RegressionSFCN(input_shape=[167, 212, 160], weights='brain-age', include_top=**False**,
dropout=0.5, weight_decay=1e-3, prediction_range=[50,105])

3.
model = RegressionSFCN(input_shape=[167, 212, 160], weights='brain-age', include_top=**True**,
dropout=0.5, weight_decay=1e-3, prediction_range=[50,105])

**Comment by estenhl** [comment]
You should use the third one :)

**Comment by Yingjuan** [comment]
Great. Thank you!
